apps/taskman/service.py

```diff
@@ -265,3 +265,19 @@ def get_task_period(
         query_list = self.query_tasks(query_list)
         query_list = self.query_updated(query_list, updated_after, updated_before)
         return self.run_query(query_list)
+
+    def add_link(self, issue_key, url, title):
```

Comment:
Why is this defined in the **taskman** when it is only used for the trackers? I assume it probably follows the pattern of the **create_comment** method which seems to me to have set a bad precedence - the commenting seems to be used for both tasks and trackers so is also not **taskman** specific. What about moving it (ideally both) to the general **JiraQuerier**? That could relieve some unnecessary dependencies between the task and tracker management - which would probably have to happen in the future anyway if we successfully migrate the tracker functionality elsewhere.